src/synology_dsm/api/download_station/__init__.py

```diff
@@ -32,17 +37,50 @@ def update(self):
                 self._tasks_by_id[task_data["id"]] = SynoDownloadTask(task_data)
 
     # Global
+    def update_info(self):
+        """Update info about the Download Station instance."""
+        self._info = self._dsm.get(self.INFO_API_KEY, "GetInfo")["data"]
+
     def get_info(self):
         """Return general informations about the Download Station instance."""
-        return self._dsm.get(self.INFO_API_KEY, "GetInfo")
+        return self._info
+
+    def update_config(self):
+        """Update configuration about the Download Station instance."""
+        self._config = self._dsm.get(self.INFO_API_KEY, "GetConfig")["data"]
```

Comment:
- this could cause KeyError, in case _dsm.get() was unsuccessful
- prior result was not restricted to content of `data`